### PR TITLE
Fix `BindException: Address already in use` in HDFS integration tests

### LINKED ISSUE #48975: test_storage_hdfs Can't wait HDFS to start
https://s3.amazonaws.com/clickhouse-test-reports/48664/61635eb2d345881a7fd84a2c2708d9d2d02c3e04/integration_tests__release__%5B3/4%5D.html

Happens not too often: [ci db](https://play.clickhouse.com/play?user=play#U0VMRUNUIGNvdW50KCkgYXMgYywgbWF4KHB1bGxfcmVxdWVzdF9udW1iZXIpLCBjaGVja19zdGFydF90aW1lLCBhbnkoY2hlY2tfbmFtZSksIHJlcG9ydF91cmwKRlJPTSBjaGVja3MKV0hFUkUgMQogICAgLS1BTkQgcHVsbF9yZXF1ZXN0X251bWJlciA9IDAKICAgIEFORCB0ZXN0X3N0YXR1cyA9ICdFUlJPUicKCiAgICBBTkQgdGVzdF9uYW1lIGxpa2UgJyV0ZXN0X3N0b3JhZ2VfaGRmcyUnCkdST1VQIEJZIHJlcG9ydF91cmwsIGNoZWNrX3N0YXJ0X3RpbWUKSEFWSU5HIGMgPiAyMApPUkRFUiBCWSBjaGVja19zdGFydF90aW1lIGRlc2M=)

### PR DESCRIPTION
**Pull request by nickitat**
### Changelog category (leave one):
- Not for changelog (changelog entry is not required)

### CODE REVIEW

**Comment by robot-ch-test-poll** [comment]
This is an automated comment for commit f43e916d42db350545d330bfb16ea57a9bd5e0d4 with description of existing statuses. It's updated for the latest CI running
The full report is available [here](https://s3.amazonaws.com/clickhouse-test-reports/49428/f43e916d42db350545d330bfb16ea57a9bd5e0d4/ci_running.html)
The overall status of the commit is 🟢 success

<table><thead><tr><th>Check name</th><th>Description</th><th>Status</th></tr></thead>
<tbody><tr><td>Style Check</td><td>Runs a set of checks to keep the code style clean. If some of tests failed, see the related log from the report.</td><td>🟢 success</td></tr>
</table>

**Comment by nickitat** [comment]
@kssenii maybe you could take a look?

**Review by alesapin — COMMENTED** [review]
0_о

File: tests/integration/test_storage_kerberized_hdfs/hdfs_configs/bootstrap.sh [hunk 1]
```diff
@@ -111,6 +111,23 @@ cat > /usr/local/hadoop/etc/hadoop/hdfs-site.xml << EOF
   <name>dfs.datanode.http.address</name>
   <value>0.0.0.0:1006</value>
 </property>
+<!-- If the port is 0 then the server will start on a free port. -->
```
Comment (kssenii) [inline tests/integration/test_storage_kerberized_hdfs/hdfs_configs/bootstrap.sh:114]: I see the linked issue https://github.com/ClickHouse/ClickHouse/issues/48975 is about `test_storage_hdfs`, but this file is related to `test_storage_kerberized_hdfs` only, so it does not fix the issue?

Comment (alexey-milovidov) [reply tests/integration/test_storage_kerberized_hdfs/hdfs_configs/bootstrap.sh:114]: @nickitat let's just copy this config change into all HDFS tests?

**Review by alexey-milovidov — APPROVED** [review]

**Pull request merged**